pkgs/top-level/python-packages.nix

```diff
@@ -23341,7 +23341,7 @@ in {
     };
   };
 
-  Theano = buildPythonPackage rec {
+  theano = buildPythonPackage rec {
```

Comment:
Within `python-packages.nix` we also aim to stick to the exact name upstream uses.